src/main/kotlin/com/coder/gateway/cli/CoderCLIManager.kt

```diff
@@ -289,7 +290,19 @@ class CoderCLIManager(
                       SetEnv CODER_SSH_SESSION_TYPE=JetBrains
                     """.trimIndent()
                         .plus(extraConfig)
-                        .replace("\n", System.lineSeparator())
+                        .plus("\n")
```

Comment:
I am not sure it really matters, but just in case I have been using `System.lineSeparator()` which I think spits out `\r\n` on Windows.

```suggestion
                        .plus(System.lineSeparator())
```

Reply:
Oh wait we do a replace further down which would also replace this one.  Ignore me!

Reply:
Yeah I had some trial and error where the windows test was producing incorrect output and the \n seemed to work well